src/net_processing.cpp

```diff
@@ -1018,6 +1019,13 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans)
     return nEvicted;
 }
 
+unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans)
+{
+    LOCK(g_cs_orphans);
+    SweepOutExpiredOrphans();
+    return EvictOrphanTxs(nMaxOrphans);
+}
```

Comment:
Why is net_processing changed in a pull that adds a benchmark? Seems unnecessary to me

Reply:
To benchmark `EvictOrphanTxs()` without sweeping out functionality of the current `LimitOrphanTxSize()`.

What could you suggest to change?

Reply:
If you explicitly want to disable the time based eviction, it could be done with mocktime in the benchmark, no?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19377#issuecomment-650013735).